### PR TITLE
docs: new readme

### PR DESCRIPTION
**Pull request by maielo**
Added brand new readme :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,99 @@
...
+- 💯&nbsp;Free & open-source.
+- 🎨&nbsp;Infinite, canvas based, hand-drawn whiteboard.
```
Comment (dwelle) [inline README.md:50]:
```suggestion
- 🎨&nbsp;Infinite, canvas-based, hand-drawn whiteboard.
```

File: README.md [hunk 2]
```diff
@@ -1,44 +1,99 @@
...
+- 🌓&nbsp;Dark mode.
+- 🏗️&nbsp;Customizable.
+- 📷&nbsp;Image support.
+- 😀&nbsp;Shape libraries support.
+- 👅&nbsp;Localization (i18n) support.
+- 🖼️&nbsp;Export to PNG, SVG & clipboard.
+- 💾&nbsp;Open format - save drawing as `.excalidraw` json file.
+- ⚒️&nbsp;Wide range of tools - Panning, rectangle, circle, diamond, arrow, line, free-draw, eraser.
```
Comment (dwelle) [inline README.md:58]:
```suggestion
- ⚒️&nbsp;Wide range of tools - rectangle, circle, diamond, arrow, line, free-draw, eraser...
```

File: README.md [hunk 3]
```diff
@@ -1,44 +1,99 @@
...
+- ➡️&nbsp;Arrow-binding & Labeled arrows.
```
Comment (dwelle) [inline README.md:59]:
```suggestion
- ➡️&nbsp;Arrow-binding & labeled arrows.
```

File: README.md [hunk 4]
```diff
@@ -1,44 +1,99 @@
...
+- 🔙&nbsp;Undo / Redo.
+- 🔍&nbsp;Zoom.
```
Comment (dwelle) [inline README.md:61]:
```suggestion
- 🔍&nbsp;Zoom and panning support
```

File: README.md [hunk 5]
```diff
@@ -1,44 +1,99 @@
...
-## Developers
+or via yarn
 
-You can integrate Excalidraw into your app by installing our [npm component](https://npmjs.com/package/@excalidraw/excalidraw).
+```
+yarn add react react-dom @excalidraw/excalidraw
+```
 
-Visit our documentation on [https://docs.excalidraw.com](https://docs.excalidraw.com).
+## Contributions
```
Comment (dwelle) [inline README.md:77]:
```suggestion
## Contributing
```

File: README.md (hunk 2 above)
Comment (dwelle) [reply README.md:58]: panning not a tool per se. I've put it below with zoom support.

Comment (maielo) [reply README.md:58]: yup makes sense

File: README.md [hunk 6]
```diff
@@ -1,44 +1,99 @@
...
+- ⚒️&nbsp;Wide range of tools - rectangle, circle, diamond, arrow, line, free-draw, eraser...
+- ➡️&nbsp;Arrow-binding & labeled arrows.
+- 🔙&nbsp;Undo / Redo.
+- 🔍&nbsp;Zoom and panning support.
 
-<a href="https://opencollective.com/excalidraw#category-CONTRIBUTE" target="_blank"><img src="https://opencollective.com/excalidraw/tiers/backers.svg?avatarHeight=32"/></a>
+## Quick start
 
-Last but not least, we're thankful to these companies for offering their services for free:
+- Don't forget to check out our [Documentation](https://docs.excalidraw.com) & [npm package](https://www.npmjs.com/package/@excalidraw/excalidraw)
```
Comment (ad1992) [inline README.md:65]: How about having an "Integration Section` and mention npm package and VsCode in that ?

File: README.md [hunk 7]
```diff
@@ -1,44 +1,99 @@
...
+    <img alt="Excalidraw" src="https://excalidraw.nyc3.cdn.digitaloceanspaces.com/github%2FExcalidraw_Github_cover.png" />
+  </picture>
+</a>
 
-## Try now
+<h4 align="center">
+  <a href="https://blog.excalidraw.com">Blog</a> |
+  <a href="https://docs.excalidraw.com">Documentation</a> |
+  <a href="https://excalidraw.com">Excalidraw Editor</a> |
+  <a href="https://plus.excalidraw.com">Excalidraw+</a>
```
Comment (dwelle) [inline README.md:12]: swap order in this way?

```
Excalidraw Editor
Documentation
Blog
Excalidraw+
```

File: README.md [hunk 8]
```diff
@@ -1,44 +1,99 @@
...
+  </a>
+</p>
 
-## Supporting Excalidraw
+<div align="center">
+  <figure>
+    <img src="https://excalidraw.nyc3.cdn.digitaloceanspaces.com/github%2Fproduct_showcase.png" alt="Product showcase" />
+    <figcaption>
+      <p align="center">
+        Create beautiful hand-drawn like diagrams, wireframes or whatever you like.
```
Comment (dwelle) [inline README.md:41]:
```suggestion
        Create beautiful hand-drawn like diagrams, wireframes, or whatever you like.
```

File: README.md (hunk 6 above)
Comment (dwelle) [inline README.md:65]:
```suggestion
Install [Excalidraw npm package](https://www.npmjs.com/package/@excalidraw/excalidraw):
```

File: README.md [hunk 9]
```diff
@@ -1,44 +1,99 @@
...
+npm install react react-dom @excalidraw/excalidraw
+```
 
-## Developers
+or via yarn
 
-You can integrate Excalidraw into your app by installing our [npm component](https://npmjs.com/package/@excalidraw/excalidraw).
+```
+yarn add react react-dom @excalidraw/excalidraw
+```
```
Comment (dwelle) [inline README.md:75]:
```suggestion
```
```

Don't forget to check out our [Documentation](https://docs.excalidraw.com)!

File: README.md (hunk 6 above)
Comment (dwelle) [reply README.md:65]: moved the docs mention below the install codeblocks.

Comment (maielo) [reply README.md:65]: Good catch :-) gonna add it to integration

File: README.md (hunk 7 above)
Comment (maielo) [reply README.md:12]: try it - not visually appealing

File: README.md (hunk 9 above)
Comment (maielo) [reply README.md:82]: what is this?

File: README.md (hunk 7 above)
Comment (dwelle) [reply README.md]: went with:

image

We have to have the editor as first link

File: README.md [hunk 10]
```diff
@@ -1,44 +1,112 @@
...
+      </p>
+    </figcaption>
+  </figure>
 </div>
 
-## Try now
+## Features
 
-Visit [excalidraw.com](https://excalidraw.com) to start sketching.
+- 🎨&nbsp;Infinite, canvas-based, online whiteboard.
```
Comment (ad1992) [inline README.md:54]: Though "Collaboration" and "End to end encryption" is mentioned above but I think we should highlight in the features too

Comment (ad1992) [reply README.md:54]: Also, we should mention `PWA` as well

File: README.md [hunk 11]
```diff
@@ -1,44 +1,112 @@
...
-You can integrate Excalidraw into your app by installing our [npm component](https://npmjs.com/package/@excalidraw/excalidraw).
-
-Visit our documentation on [https://docs.excalidraw.com](https://docs.excalidraw.com).
+- [VScode extension](https://marketplace.visualstudio.com/items?itemName=pomdtr.excalidraw-editor)
+- [npm package](https://www.npmjs.com/package/@excalidraw/excalidraw)
 
 ## Who's integrating Excalidraw
 
-[Google Cloud](https://googlecloudcheatsheet.withgoogle.com/architecture) • [Meta](https://meta.com/) • [CodeSandbox](https://codesandbox.io/) • [Obsidian Excalidraw](https://github.com/zsviczian/obsidian-excalidraw-plugin) • [Replit](https://replit.com/) • [Slite](https://slite.com/) • [Notion](https://notion.so/) • [HackerRank](https://www.hackerrank.com/)
+[Google Cloud](https://googlecloudcheatsheet.withgoogle.com/architecture) • [Meta](https://meta.com/) • [CodeSandbox](https://codesandbox.io/) • [Obsidian Excalidraw](https://github.com/zsviczian/obsidian-excalidraw-plugin) • [Replit](https://replit.com/) • [Slite](https://slite.com/) • [Notion](https://notion.so/) • [HackerRank](https://www.hackerrank.com/) • and many others
```
Comment (ad1992) [inline README.md:98]: can we beautify this by using logos instead of just names?

File: README.md (hunk 10 above)
Comment (dwelle) [reply README.md:54]: neither applies to the package which is the readme's target

File: README.md (hunk 11 above)
Comment (dwelle) [reply README.md:112]: TBD later

**Comment by dwelle** [comment]
🚀

**Review by dwelle — APPROVED** [review]

**Pull request merged**